web_console/libs/es.js

```diff
@@ -7,64 +7,71 @@ const ky = require('ky-universal');
 const getConfig = require('../utils/get_confg');
 
 const config = getConfig({
-    ES_HOST: process.env.ES_HOST,
-    ES_PORT: process.env.ES_PORT,
+  ES_HOST: process.env.ES_HOST,
+  ES_PORT: process.env.ES_PORT,
 });
 
-
 class ElasticSearchClient {
-    constructor() {
-        // TODO: use HTTPs for production
-        const prefixUrl = `http://${config.ES_HOST}:${config.ES_PORT}`;
-        this.client = ky.create({ prefixUrl });
-    }
+  constructor() {
+    // TODO: use HTTPs for production
+    const prefixUrl = `http://${config.ES_HOST}:${config.ES_PORT}`;
+    this.client = ky.create({ prefixUrl });
+  }
 
-    async queryLog(index, keyword, pod_name, start_time, end_time) {
-        let query_body = {
-            "sort": [
-                {
-                    "@timestamp": {
-                        "order": "asc",
-                        "unmapped_type": "boolean"
-                    }
+  async queryLog(index, keyword, pod_name, start_time, end_time, match_phrase) {
```

Comment:
only support `index` and `options/dsl` arguments would be better, and implement a utility function for `es` dsl generation